IFComp/t/entry_processing.t

```diff
@@ -0,0 +1,120 @@
+use strict;
+use warnings;
+use Test::More;
+
+use FindBin;
+use lib ("$FindBin::Bin/lib");
+use IFCompTest;
+my $schema = IFCompTest->init_schema();
+
+my $entry_directory =
+    Path::Class::Dir->new("$FindBin::Bin/platform_test_entries");
+
+$schema->entry_directory($entry_directory);
+
+$schema->populate(
+    'Entry',
+    [   [ 'id', 'author', 'title',                   'comp' ],
+        [ 100,  1,        'Test Z-code game',        1 ],
+        [ 101,  1,        'Test Glulx game',         1 ],
+        [ 102,  1,        'Test Quixe game',         1 ],
+        [ 103,  1,        'Test Parchment game',     1 ],
+        [ 104,  1,        'Test Z-code website',     1 ],
+        [ 105,  1,        'Test non-Inform website', 1 ],
+        [ 106,  1,        'Test HTML page',          1 ],
+        [ 107, 1, 'Test Z-code website with buried story file', 1 ],
+        [ 108, 1, 'Test Quest game',                            1 ],
+        [ 109, 1, 'Test TADS game',                             1 ],
+        [ 110, 1, 'Test Alan game',                             1 ],
+    ],
+);
+
+for my $entry ( $schema->resultset('Entry')->all ) {
+    $entry->update_content_directory;
+}
+
+sub file_exists ($$) {
+    my ( $entry_id, $filename ) = @_;
+    return -e "$entry_directory/$entry_id/content/$filename";
+}
+
+sub file_contains ($$$) {
+    my ( $entry_id, $filename, $regex ) = @_;
+    my $file = Path::Class::File->new(
+        "$entry_directory/$entry_id/content/$filename" );
+    eval { return $file->slurp =~ $regex; };
+}
+
+diag('Testing naked Z-Code upload...');
```

Comment:
Could you change the diag() lines to note()? The only difference is that note() won't display when running under a harness, like prove.